src/app/tests/integration/chip_im_responder.cpp

```diff
@@ -50,6 +50,25 @@
 namespace chip {
 namespace app {
 
+namespace {
+
+class TestTLVDataEncoder : public chip::app::DataModel::EncodableToTLV
```

Comment:
```suggestion
class TestTLVDataEncoder : public DataModel::EncodableToTLV
```
This is all inside chip::app.